internal/services/mobilenetwork/mobile_network_slice_resource.go

```diff
@@ -190,7 +190,7 @@ func (r SliceResource) Update() sdk.ResourceFunc {
 				}
 			}
 
-			if metadata.ResourceData.HasChange("snssai") {
+			if metadata.ResourceData.HasChange("single_network_slice_selection_assistance_information") {
```

Comment:
could we include an update of this property in the tests?

Reply:
test added